### ISSUE TITLE
[A11y]: Element does not have inner text that is visible to screen readers

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/whiteboard#892]
<img width="450" height="309" alt="Image" src="https://github.com/user-attachments/assets/48af8b34-38ed-4d55-9723-03fce850e04e" />

Menu toggle element should have explicit <label> element to be visible for screen readers.
